components/items/ItemsGrid/ItemsGridImage.vue

```diff
@@ -98,6 +128,27 @@ const onClickShoppingCart = () => {
     width: unset !important;
   }
 }
+
+.override-button-loading.o-btn--loading {
+  @include ktheme() {
+    color: theme('k-grey') !important;
+
+    :deep .o-btn__label {
+      margin: 0 3px !important;
+    }
+
+    &::after {
+      position: relative !important;
+      left: 0px !important ;
+      border: 2px solid theme('k-grey') !important;
+      border-radius: 9999px !important;
+      border-right-color: transparent !important;
+      border-top-color: transparent !important;
+      margin: 0 3px;
+    }
+  }
+}
```

Comment:
this binds this component to oruga implementation details
add a class / prop to NeoButton and use that instead